autometa/config/user.py

```diff
@@ -295,17 +302,39 @@ def run_binning(self, mgargs):
             index=True,
             header=True)
 
-def main(args):
-    logger.info(args.user)
-
-if __name__ == '__main__':
+def main():
     import argparse
+    import multiprocessing as mp
     import logging as logger
+    parser = argparse.ArgumentParser(description="""
+    Configures the Autometa user environment/databases.
+    Running without args will download and format Autometa database dependencies.
+    """)
+    parser.add_argument('--config',
+        help=f'</path/to/user.config> (default is {config.DEFAULT_FPATH}).',
```

Comment:
As this is a user-facing help message, should be more friendly. For example "Path to an Autometa .config file". Question - can more than one be specified? If so, should state explicitly in the help message so that people know.

Reply:
No, this will be the base config. Separate from an input metagenome form

Reply:
The `nargs='*'` argument in `add_argument` will tell argparse to collect multiple inputs. The reasoning behind `nargs='*'` (0 or more required) rather than `nargs='+'` (1 or more required) is due to the user having the option of not passing any metagenome.config file because they want to check their dependencies.